README.md

```diff
@@ -2,8 +2,25 @@
 
 This is an adapter add-on for the [Mozilla WebThings Gateway](https://github.com/mozilla-iot/gateway) that allows a user to discover native web things on their network.
 
-## Adding Web Things to Gateway
+## m2ag-labs fork
```

Comment:
Please remove this info, as it's no longer a fork. 😃

Reply:
Should I add some usage info for the jwt functionality?

Reply:
What do we need to add here to finish this up? I took out my stuff but have not committed it yet.

Reply:
Maybe just a simple step-by-step explanation of how to set up a device which requires authentication, e.g. navigate to Settings -> Add-ons in the UI, ...

Reply:
Will this be rev 4.9? I can update the manifest.

Reply:
It’ll end up being `0.5.0`, after I finish up the basic/digest auth bits. If you want to update the manifest, go ahead and update package.json as well.

Reply:
Ok -- see what you think of it now.